app/model/command/CreateTagCommand.scala

```diff
@@ -120,9 +121,14 @@ case class CreateTagCommand(
 
     val createdSectionId = if(createMicrosite) { sectionId } else { None }
 
+    val tagSubType: Option[String] =  `type` match {
```

Comment:
do we need to match on `type` here? Or can we do:
`trackingInformation.map(_.trackingType) orElse campaignInformation.map(_.campaignType)`

Reply:
I think what's there communicates intent more clearly